torch/ao/quantization/experimental/APoT_tensor.py

```diff
@@ -1,13 +1,66 @@
 import torch
+import enum
 from torch import Tensor
+from torch.ao.quantization.experimental.observer import APoTObserver, float_to_apot, float_to_reduced_precision
+
+# enum to represent APoT representation
+class APoTRepr(enum.Enum):
+    level_indices = 1
+    reduced_precision_fp = 2
```

Comment:
since there are only 2 known representations of APoT, we could just have a boolean flag instead of an enum. maybe 
```
use_int_repr
```